host/dynamicconfig.go

```diff
@@ -151,7 +145,7 @@ func (d *dynamicClient) GetDurationValue(
 		}
 	}
 	d.RUnlock()
-	return d.client.GetDurationValue(name, filters, defaultValue)
+	return d.client.GetDurationValue(name, filters)
 }
 
 func (d *dynamicClient) UpdateValue(name dynamicconfig.Key, value interface{}) error {
```

Comment:
may also be missing type checks?  though with a "deferred to other system" impl it's a bit more vague.

Reply:
the value parameter of UpdateValue is more generic, and the validation logic also depends on the implementation of client (see config_store_client), so we cannot do checks here